mailer/__init__.py

```diff
@@ -31,7 +31,8 @@ def send_mail(subject, message, from_email, recipient_list, priority='medium', f
         from_email=from_email,
         to=recipient_list,
         attachments=attachments,
-        headers=headers
+        headers=headers,
```

Comment:
Tak koukám na greception a tam jsem to vyřešil tak, že jsem upravil headers:

headers['Reply-To'] = reply_to